fontforgeexe/fontview.c

```diff
@@ -33,6 +33,7 @@
 #include "autowidth2.h"
 #include "bitmapchar.h"
 #include "bvedit.h"
+#include "charview_private.h"
```

Comment:
This header is exclusive for `cvpalettes.c`, please put `CVDrawSplinePointList()` into `views.h`

Reply:
done